pennylane/vqe/vqe.py

```diff
@@ -527,7 +529,18 @@ def metric_tensor(self, args, kwargs=None, diag_approx=False, only_construct=Fal
                 "optimized observables. Set the argument optimize=False to obtain "
                 "the metric tensor."
             )
-        # We know that for VQE, all the qnodes share the same ansatz so we select the first
+        # all the qnodes share the same ansatz so we select the first
         return self.qnodes.qnodes[0].metric_tensor(
             args=args, kwargs=kwargs, diag_approx=diag_approx, only_construct=only_construct
         )
+
+
+# This class is deprecated and does not appear in the documentation
```

Comment:
Maybe I missed it, but how did you stop it from showing in the documentation?

also, _should_ it show in the documentation? Perhaps a very short docstring with a a big `.. warning::` box explaining it the deprecation and linking to `ExpvalCost`?

Reply:
Awesome, added in again: cbf89502dcdec0c0f79585afcedf03a44bb207b8

![image](https://user-images.githubusercontent.com/49409390/99718536-03e9de80-2a79-11eb-82f0-f2ad53151df0.png)